### PR TITLE
add easyblock for DIRAC

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyblocks/d/dirac.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+    def test_step(self):
+        """Custom built-in test procedure for DIRAC."""
+        if self.cfg['runtest']:
+            # set up test environment
+            # see http://diracprogram.org/doc/release-14/installation/testing.html
+            env.setvar('DIRAC_TMPDIR', tempfile.mkdtemp(prefix='dirac-test-'))
+            env.setvar('DIRAC_MPI_COMMAND', self.toolchain.mpi_cmd_for('', self.cfg['parallel']))
+
+            # run tests (may take a while, especially if some tests take a while to time out)
```
Comment (wpoely86) [inline easybuild/easyblocks/d/dirac.py]: maybe add a warning/info about this?

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyblocks/d/dirac.py (hunk 1 above)
Comment (boegel) [inline easybuild/easyblocks/d/dirac.py]: you'll see the 'make test' log command?

Comment (wpoely86) [inline easybuild/easyblocks/d/dirac.py]: well, you don't know anything about the time? If it uses a timeout 1500s?

Comment (boegel) [inline easybuild/easyblocks/d/dirac.py]: ok, I'll add an extra info log msg

**Comment by boegel** [comment]
works well now, cfr., going in, thanks for the review @wpoely86!

**Pull request merged**